src/main/java/run/halo/app/cache/AbstractCacheStore.java

```diff
@@ -19,6 +20,8 @@
 @Slf4j
 public abstract class AbstractCacheStore<K, V> implements CacheStore<K, V> {
 
+    protected HaloProperties haloProperties;
```

Comment:
Maybe here should add final modifier after protected.

Reply:
if we want to force the constructors to pass value, it could be. 
"InMemoryCache" store implementation is not dependent to haloProperties.